src/app/item-review/destiny-tracker.service.ts

```diff
@@ -44,6 +44,8 @@ export interface DtrUserReview {
   isIgnored?: boolean;
   selectedPerks: number[];
   attachedMods: number[];
+  mode: number;  // play mode; corresponds to DestinyActivityModeType
```

Comment:
Should this be `DestinyActivityModeType` then?

Reply:
No; this is data that the DTR API returns.

Which is a number that corresponds to a `modeType` in `DestinyActivityModeType`, but if you look at the values in the manifest, that's not going to get you a unique entry.